lambda/rest-endpoints/src/app.ts

```diff
@@ -65,4 +66,37 @@ router.post('/api/curation/:region/:variant', (req: Request<CurationParams>, res
   }
 });
 
+interface RecipeIdParams {
+  immutableId: string;
+}
+
+function validateComposerParams(params: RecipeIdParams) {
+  const checker = /^[\w\d]+$/;
+
+  if(!params.immutableId.match(checker)) {
+    throw new Error("Invalid composer ID");
+  }
+}
+
+router.get('/api/content/by-uid/:immutableId', (req: Request<RecipeIdParams>, resp) => {
+  try {
+    validateComposerParams(req.params);
+  } catch(e) {
+    console.log("Provided params ", req.params, " did not validate");
+    resp.status(400).json({status: "error", detail: "invalid recipe immutable ID"});
+    return;
+  }
+
+  recipeByUID(req.params.immutableId).then(result=>{
+    if(result) {
+      console.log(`Debug: Query for ${req.params.immutableId} returned ${JSON.stringify(result)}`);
```

Comment:
Does using `console.debug` work here, and is the behaviour any different?

Reply:
I'll drop the line. `console.debug` does work, although if I remember correclty the behaviour is exactly the same.